octopus/dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM octopusdeploy/octopusdeploy:2024.1
```

Comment:
non-blocking, I would _love_ a docker compose file for octopus in the repo so I can easily launch an instance locally for testing. Maybe I'll add one during the pipeline work. :smiley:

Reply:
Feel free! One small catch, though...if you do that, you'll have to tie into the Azure database instance for this, and the system may detect it as a separate server node. That could introduce issues with licensing, etc. I'm not saying not to! I wholeheartedly support the idea if you think it prudent. Just wanted to surface the potential gotchas.